pkg/agent/keys.go

```diff
@@ -0,0 +1,105 @@
+package agent
+
+import (
+	"crypto"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/agent/keys"
+	"go.etcd.io/bbolt"
+)
+
+type keyInt interface {
+	crypto.Signer
+	//Type() string // Not Yet Supported by Krypto
+}
+
+var Keys keyInt = keys.Noop
+var LocalDbKeys keyInt = keys.Noop
+
+func SetupKeys(logger log.Logger, db *bbolt.DB) error {
+	var err error
```

Comment:
we the err var declaration? should be able to initialize on like 25?

Reply:
Line 25 is assigning directly to the package level `LocalDbKeys`. So it seems cleaner to pre-declare `err`